test/releaseTests/offlinePackage.test.ts

```diff
@@ -0,0 +1,40 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+
+import * as chai from 'chai';
+import * as glob from 'glob-promise';
+import * as path from 'path';
+import { invokeCommand } from './testAssets/testAssets';
+import { PlatformInformation } from '../../src/platform';
+import * as fs from 'async-file';
+
+suite("Offline packaging of VSIX", function () {
+    let vsixFiles: string[];
+    this.timeout(2000000);
```

Comment:
If this timeout is in milliseconds, that means this test takes nearly 30 minutes to run in travis? Probably not desirable to run that on every build?